pkg/plugin/runtime_subprocess.go

```diff
@@ -257,21 +266,47 @@ func unmarshalRuntimeConfigSubprocess(runtimeData map[string]interface{}) (*Runt
 	return &config, nil
 }
 
-// ExecDownloader executes a plugin downloader command with custom environment
-func ExecDownloader(_ string, command string, argv []string, env []string) (*bytes.Buffer, error) {
-	prog := exec.Command(command, argv...)
-	prog.Env = env
-
-	buf := bytes.NewBuffer(nil)
-	prog.Stdout = buf
-	prog.Stderr = os.Stderr
-
+func executeCmd(prog *exec.Cmd, pluginName string) error {
```

Comment:
since we've stopped passing env here, we no longer set `prog.Env = env` before `prog.Run()`. Is this handled another way somewhere?